pkgs/test/test/runner/runner_test.dart

```diff
@@ -698,4 +703,108 @@ void main(List<String> args) async {
       await test.shouldExit(0);
     });
   });
+
+  group('nnbd', () {
+    final _testContents = '''
+import 'package:test/test.dart';
+import 'weak.dart';
+
+void main() {
+  test("success", () {
+    expect(foo, true);
+  });
+}''';
+
+    setUp(() async {
+      await d.file('weak.dart', '''
+// @dart=2.8
+final foo = true;''').create();
+    });
+
+    test('strong mode is used if the entrypoint opts in explicitly', () async {
+      await d.file('test.dart', '''
+// @dart=2.9
+$_testContents
+''').create();
+      var test = await runTest(['test.dart']);
+
+      expect(
+          test.stdout,
+          containsInOrder([
+            'Unable to spawn isolate:',
+            'Error: A library can\'t opt out of non-nullable by default, when in nnbd-strong mode.'
```

Comment:
Is this our message, or is it coming from somewhere else?

cc @kevmoo - how important is it that our user facing messages use "sound" instead of "strong"?

Reply:
This is likely originating from the CFE.